.github/workflows/ingest-cloud.yml

```diff
@@ -31,6 +31,20 @@ jobs:
     - name: GCP Credential Workaround
       run: |
         echo '{ "type": "service_account" }' > .dummy_adc.json
+    
+    # Setup v2 credentials for accessing AWS S3 buckets. Each set of credentials
+    # is given its own profile which is then set in the config using the
+    # "profile=PROFILENAME" query parameter in the source's bucket URL.
+    - name: AWS Credentials
+      run: |
+        mkdir "$HOME/.aws"
+        echo '[reversinglabs]' > "$HOME/.aws/credentials"
+        echo "aws_access_key_id = $RL_AWS_ACCESS_KEY_ID" >> "$HOME/.aws/credentials"
+        echo "aws_secret_access_key = $RL_AWS_SECRET_ACCESS_KEY" >> "$HOME/.aws/credentials"
+        echo '[profile reversinglabs]' > $HOME/.aws/config"
```

Comment:
Is there any concern about credential leaking? I know GitHub will hide this in the logs, but this goes against their recommendations:
https://docs.github.com/actions/security-guides/using-secrets-in-github-actions#using-secrets-in-a-workflow


> Avoid passing secrets between processes from the command line, whenever possible. Command-line processes may be visible to other users (using the ps command) or captured by [security audit events](https://docs.microsoft.com/windows-server/identity/ad-ds/manage/component-updates/command-line-process-auditing). To help protect secrets, consider using environment variables, STDIN, or other mechanisms supported by the target process.

Reply:
Fixed. Now uses `envsubst` to generate the file.